config/config.schema.json

```diff
@@ -741,55 +743,11 @@
           "softDelete"
         ]
       },
-      "subnetPrefix": {
-        "type": "string"
-      },
       "svc": {
         "type": "object",
         "properties": {
-          "etcd": {
-            "type": "object",
-            "properties": {
-              "kvName": {
-                "type": "string"
-              },
-              "kvSoftDelete": {
-                "type": "boolean"
-              }
-            },
-            "additionalProperties": false,
-            "required": [
-              "kvName",
-              "kvSoftDelete"
-            ]
-          },
-          "userAgentPool": {
-            "type": "object",
-            "properties": {
-              "azCount": {
-                "type": "number"
-              },
-              "maxCount": {
-                "type": "number"
-              },
-              "minCount": {
-                "type": "number"
-              },
-              "osDiskSizeGB": {
-                "type": "number"
-              },
-              "vmSize": {
-                "type": "string"
-              }
-            },
-            "additionalProperties": false,
-            "required": [
-              "azCount",
-              "maxCount",
-              "minCount",
-              "osDiskSizeGB",
-              "vmSize"
-            ]
+          "aks": {
+            "$ref": "#/definitions/aksConfig"
```

Comment:
Do you have future use of aksConfig in mind? I really like this approach for node pools, but I think it could be overused in this particular case.

Reply:
It's in use twice in the config, once for SC and once for the MC. Right now, both of them are the same so I thought reuse makes sense. I'm open to suggestions though.